cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/vertx/generator/AnnotationMappers.java

```diff
@@ -0,0 +1,244 @@
+/*
```

Comment:
If these classes in the `generator` package are taken from somewhere then we should use the original license and also we should note it in the NOTICE file.